webapp/publisher/snaps/build_views.py

```diff
@@ -247,7 +247,7 @@ def get_snap_builds_json(snap_name):
 
     start = flask.request.args.get("start", 0, type=int)
     size = flask.request.args.get("size", 15, type=int)
-    build_slice = slice(start, start + size)
+    build_slice = slice(start, size)
```

Comment:
Is this not needed? The two cases from the frontend:
1) Clicking show more: start = 15, size = 15
2) Refreshing the status of all visible rows (after clicking show more 30+) which means: start = 0, size = 30+

Reply:
Hey @Lukewh , it doesn't match with my experience, this are the requests made in the frontend:

Show more: builds.json?start=15&size=30
Table refresh: builds.json?size=30

Reply:
I just QA'd and it works fine - no sure what the `start + ` was for :shrug: